docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -127,7 +127,7 @@ for this: `publish(<from-stage>, <to-stage>)`.
 	// and write a row to `MyRecord_versions`.
 	$record->write(); 
 	// will copy the saved record information to the `MyRecord_Live` table
-	$record->publish('Stage', 'Live');
+	$record->copyVersionToStage(Versioned::DRAFT, Versioned::LIVE);
```

Comment:
This doc change is incomplete (the publish method is still referenced a few lines above).